### PR TITLE
Update figure card to have title displayed up

### PR DESCRIPTION
**Pull request by TanMar23**
## Description
Update figure card to have title displayed up

## Testing
- npm run test: *pass*
- npm run lint: *pass*

## Screenshots/Captures
title-up-fg

## Checks
- [ ] Requires documentation update
- [ ] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [ ] **MINOR** adds functionality in a backwards-compatible manner.
- [x] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: src/components/FigureCard/FigureCard.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ const props = {
     type: String,
     default: '',
   },
+  isTitleUp: {
```
Comment (jasonadt) [inline src/components/FigureCard/FigureCard.js]: (nit) please rename this to: `titleAbove`

File: src/components/FigureCard/FigureCard.vue [hunk 2]
```diff
@@ -1,12 +1,16 @@
 <template>
   <figure class="figure-card">
-    <div class="image-wrapper">
+    <figcaption v-show="isTitleUp" v-if="title" :data-test-id="`${dataTestId}-title`">
```
Comment (jasonadt) [inline src/components/FigureCard/FigureCard.vue:3]: Please remove the `v-show` attribute in favor of updating `v-if`:

```js
<figcaption v-if="titleAbove && title" :data-test-id="`${dataTestId}-title`">
```

File: src/components/FigureCard/FigureCard.vue [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+    </figcaption>
+    <div v-if="imageSource" class="image-wrapper">
       <div class="image"
            :data-testid="`${dataTestId}-image`"
            :style="backgroundStyle"
       />
     </div>
-    <figcaption v-if="title" :data-test-id="`${dataTestId}-title`">
+    <slot/>
+    <figcaption v-show="!isTitleUp" v-if="title" :data-test-id="`${dataTestId}-title`">
```
Comment (jasonadt) [inline src/components/FigureCard/FigureCard.vue:13]: Similar comment here: 

```js
<figcaption v-if="!titleAbove && title" :data-test-id="`${dataTestId}-title`">
```

**Review by jasonadt — CHANGES_REQUESTED** [review]
Nice work! I've left a few small comments to address before I'll approve.

File: src/components/FigureCard/FigureCard.vue (hunk 2 above)
Comment (TanMar23) [reply src/components/FigureCard/FigureCard.vue:3]: Nice! Thanks for the recommendation

**Review by jasonadt — APPROVED** [review]
👍  ship it! 🚀

**Pull request merged**